specs/experimental/plasma.md

```diff
@@ -32,12 +32,13 @@ chain derivation must be reset, omitting the input data when rederiving therefor
 
 [vitalikblog]: https://vitalik.eth.limo/general/2023/11/14/neoplasma.html
 
-## DA Storage
+## DA Server
 
-Input data is uploaded to the storage layer via plain HTTP calls to the DA storage service.
-This service is horizontally scalable and concerned with replicating the data across preferred storage layers
-such as IPFS or any S3 compatible storage. Input data is content addressed by its hash in the request url
-so responses are easily cachable.
+Input data is uploaded to the storage layer via plain HTTP calls to the DA server.
+
+This service is responsible to interacting with the Data Availability Layer (DA layer).
+The layer could be a content addressable storage layer like IPFS or any S3 compatible storage.
+These layers use `commitment_type 0` and can pre-compute the commitment prior to posting the data.
```

Comment:
We haven't yet defined the commitment types if reading from top of doc

Reply:
Perhaps worth breaking out sub headers for the different commitment types and have specific information under each sub header

Reply:
Would linking forwards also work here? It may also make sense to reflow the entire docs to first talk about commitments, then the DA server, then derivation b/c each builds on the last

Reply:
I think your idea about changing the order makes the most sense

Reply:
Up to you if you want to do that as part of this PR

Reply:
Ah i see it was just a small diff